### DISCUSSION TITLE
Add more "sort by" options to the Products block, starting with menu_order

### DISCUSSION THREAD

**Discussion by asafm7** [woocommerce/woocommerce#45076] [WooCommerce, Blocks, and Site Editing]
Add more "sort by" options to the Products block, starting with menu_order (which used to be the default sorting in WooCommerce, if I'm not wrong).

**Comment by scrobbleme** [comment]
It is really annoying that is not possible in the blocks anymore...

**Comment by kmanijak** [comment]
We'll add this option in Product Collection block!

Here's the issue that we aim to address for WooCommerce 9.5: https://github.com/woocommerce/woocommerce/issues/49529. Thanks for raising this topic! In the meantime I'll go ahead and close this discussion and let's continue in the issue if needed. 🙌
